### PR TITLE
302: edit functionality on FileDocument form from review page

### PR DESCRIPTION
**Pull request by rachaelparris**

### CODE REVIEW

File: web-client/src/views/FileDocument/PrimaryDocumentForm.jsx [hunk 1]
```diff
@@ -489,19 +493,10 @@ export const PrimaryDocumentForm = connect(
                       <FontAwesomeIcon icon="check-circle" size="sm" />
                     </span>
                   </label>
-                  <input
+
+                  <StateDrivenFileInput
                     id="supporting-document-file"
-                    type="file"
-                    accept=".pdf"
-                    aria-describedby="supporting-document-file-label"
```
Comment (kkoskelin) [inline web-client/src/views/FileDocument/PrimaryDocumentForm.jsx:496]: Could we restore the aria-describedby attributes everywhere, allow them to "pass through"?

File: web-client/src/views/FileDocument/SecondaryDocumentForm.jsx [hunk 2]
```diff
@@ -66,20 +68,12 @@ export const SecondaryDocumentForm = connect(
                     <span className="usa-form-hint">(optional)</span>
                   )}
                 </label>
-                <input
+
+                <StateDrivenFileInput
                   id="secondary-document"
-                  type="file"
-                  accept=".pdf"
-                  aria-describedby="secondary-document-label"
```
Comment (kkoskelin) [inline web-client/src/views/FileDocument/SecondaryDocumentForm.jsx:73]: restore aria-describedby, please

File: web-client/src/views/FileDocument/SecondaryDocumentForm.jsx [hunk 3]
```diff
@@ -251,20 +249,12 @@ export const SecondaryDocumentForm = connect(
                       <FontAwesomeIcon icon="check-circle" size="sm" />
                     </span>
                   </label>
-                  <input
+
+                  <StateDrivenFileInput
                     id="secondary-supporting-document-file"
-                    type="file"
-                    accept=".pdf"
-                    aria-describedby="secondary-supporting-document-file-label"
```
Comment (kkoskelin) [inline web-client/src/views/FileDocument/SecondaryDocumentForm.jsx:258]: restore

**Review by kkoskelin — CHANGES_REQUESTED** [review]
Just some accessibility things to address, but nice.

**Review by codyseibert — APPROVED** [review]

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**